dango/dex/src/cron.rs

```diff
@@ -539,6 +544,31 @@ fn clear_orders_of_pair(
             volumes,
             volumes_by_username,
         )?;
+
+        #[cfg(feature = "metrics")]
+        {
+            metric_volume
+                .entry((&base_denom, &quote_denom, &base_denom))
+                .or_insert(grug::Int::ZERO)
+                .checked_add_assign(filled_base.into_int_floor())?;
+            metric_volume
+                .entry((&base_denom, &quote_denom, &quote_denom))
+                .or_insert(grug::Int::ZERO)
+                .checked_add_assign(filled_quote.into_int_floor())?;
```

Comment:
You don't need to have a temporary `metric_volume` HashMap, just use `metrics::histogram!` here. You should also add description about those metrics, look at `init_metrics` and `describe_histogram!`. You only need to call those describes once at startup time.